### ISSUE TITLE
Not all twisted._threads modules are installed on py3

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7990]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)| @hawkowl reported|
|-|-|
|Trac ID|trac#7990|
|Type|release blocker: regression|
|Created|2015-08-07 09:24:44Z|
|Branch|https://github.com/twisted/twisted/tree/threads-py3-7990|

```
Traceback (most recent call last):
  File "/vagrant/testresolve.py", line 7, in <module>
    reactor.resolve('localhost').addCallback(done)
  File "/home/vagrant/pyenv3/lib/python3.4/site-packages/twisted/internet/base.py", line 570, in resolve
    return self.resolver.getHostByName(name, timeout)
  File "/home/vagrant/pyenv3/lib/python3.4/site-packages/twisted/internet/base.py", line 271, in getHostByName
    self.reactor, self.reactor.getThreadPool(),
  File "/home/vagrant/pyenv3/lib/python3.4/site-packages/twisted/internet/base.py", line 991, in getThreadPool
    self._initThreadPool()
  File "/home/vagrant/pyenv3/lib/python3.4/site-packages/twisted/internet/base.py", line 956, in _initThreadPool
    from twisted.python import threadpool
  File "/home/vagrant/pyenv3/lib/python3.4/site-packages/twisted/python/threadpool.py", line 16, in <module>
    from twisted._threads import pool as _pool
  File "/home/vagrant/pyenv3/lib/python3.4/site-packages/twisted/_threads/__init__.py", line 15, in <module>
    from ._pool import pool
ImportError: No module named 'twisted._threads._pool'
```

<details><summary>Searchable metadata</summary>

```
trac-id__7990 7990
type__release_blocker__regression release blocker: regression
reporter__hawkowl hawkowl
priority__normal normal
milestone__None None
branch__branches_threads_py3_7990 branches/threads-py3-7990
branch_author__hawkowl hawkowl
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1438939484623113 1438939484623113
changetime__1445217443410268 1445217443410268
version__None None
owner__hawkowl hawkowl

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_1"></a>|@hawkowl commented|
|-|-|

(In [45399]) Branching to threads-py3-7990.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_2"></a>|@hawkowl commented|
|-|-|

Reproduction script from https://github.com/twisted/txmongo/pull/124#issuecomment-128622091 :

```
from twisted.internet import reactor 

def done(result):
    print('done', result)
    reactor.stop()

reactor.resolve('localhost').addCallback(done)
reactor.run()
```

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_3"></a>|@hawkowl commented|
|-|-|

This should fix it... builders look green.

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_5"></a>|@glyph set owner to @hawkowl|
|-|-|

I'm unhappy that a builder was not red first.  We really need more builders to test Twisted-as-installed.  But the fix looks correct so please land.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_6"></a>|@hawkowl set status to `closed`|
|-|-|

(In [45415]) Merge threads-py3-7990: Install all _threads submodules on Python 3

Author: hawkowl
Reviewer: glyph
Fixes: [#7990](https://github.com/twisted/twisted/issues/7990)

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_7"></a>|@hawkowl commented|
|-|-|

Ticket retargeted after milestone deleted
